Products/ZenCallHome/CallHomeStatus.py

```diff
@@ -76,10 +76,9 @@ def load_from_redis(self, k="CallHomeStatus"):
             log.warning(
                 "Exception trying to recive data from redis: {0}".format(e))
             self._redis_client = None
-            return
+            return dict()
```

Comment:
I haven't examined all of the code, but I suspect that usages of `push_to_redis` and `load_from_redis` can be simplified if those methods handled object pickling themselves.  I.e. Move all the calls to `pickle.dumps` into `push_to_redis` and the calls to `pickle.loads` into `load_from_redis`.